openedx/features/redhouse_features/registration/forms.py

```diff
@@ -0,0 +1,33 @@
+"""
+Redhouse registration application form
+"""
+from django import forms
+
+from openedx.features.redhouse_features.registration.models import AdditionalRegistrationFields
+
+
+class RedhouseUserProfile(forms.ModelForm):
```

Comment:
Please rename it back to `AdditionalRegistrationFieldsForm`

```suggestion
class AdditionalRegistrationFieldsForm(forms.ModelForm):
```